.travis.yml

```diff
@@ -49,9 +52,16 @@ matrix:
       env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
     - php: hhvm
       env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
+    - php: hhvm
+      env: DB=pgsql POSTGRESQL_VERSION=9.4 # driver currently unsupported by HHVM
```

Comment:
Wondering if this still holds true considering https://github.com/PocketRent/hhvm-pgsql. Is HHVM really not supporting a single actually working RDBMS?